stdlib/public/Darwin/Dispatch/Block.swift

```diff
@@ -97,6 +98,10 @@ public class DispatchWorkItem {
 	public var isCancelled: Bool {
 		return _swift_dispatch_block_testcancel(_block) != 0
 	}
+
+	public func dynamicallyCall(withArguments: [()]) -> Void {
```

Comment:
New members either need an availability declaration (spelled `@available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)`), or they need to be declared `@_alwaysEmitIntoClient`.

Since this is a forwarding method that exists for convenience only, it makes sense to make it back-deployable to all runtime environments:

```
  @_alwaysEmitIntoClient
  public func callAsFunction() { ... }
```

(This embeds the body of the function in clients that use it, so you won't be able to update it later. It's like a `static inline` function in a header.)

Stylistic note: the `-> Void` is implicit in this context, and it is usually omitted.